x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts

```diff
@@ -369,7 +369,7 @@ export const markAsFavorite = () => {
 };
 
 export const openTimelineFieldsBrowser = () => {
-  cy.get(TIMELINE_FIELDS_BUTTON).first().click({ force: true });
+  cy.get(TIMELINE_FIELDS_BUTTON).first().click();
```

Comment:
I just want to note that the `first()` here is pretty dangerous/brittle. I was trying to remove it and realized that we have 3 instances of the `FieldBrowserComponent` which contains the `EuiButtonEmpty` that has the `data-test-subj="show-field-browser"` property.
It happens that in the DOM the first one is the one that we mostly want, but that could change.

I don't think this PR is the right place to do it, so this is just a note, but we should at some point modify the `FieldBrowserComponent` to take the `data-test-subj- as a prop so we can guarantee the uniqueness.

Reply:
Agreed, `first()` often is a source of flakiness/trouble in the future. I wouldn't want to touch it in this PR. Also, it seems like the component is owned by ResponseOps and not us...